mlir/lib/Dialect/Rock/utility/loweringUtils.cpp

```diff
@@ -19,6 +19,7 @@
 #include "mlir/Dialect/Rock/IR/Rock.h"
 #include "mlir/Dialect/Rock/Tuning/ConvContext.h"
 #include "mlir/Dialect/Rock/utility/math.h"
+#include "mlir/Dialect/SCF/IR/SCF.h"
```

Comment:
nit:

These may not be necessary to include if it is not being used in this source file